src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="container">
+        <legend>Personal information</legend>
+
+        <label class="form_field">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
```

Comment:
```suggestion
            name="surname"
```
use camelCase for value of name attribute 
4 point in checklist, make sure you fix it everywhere not just here